angular-client/src/pages/landing-page/landing-page.component.html

```diff
@@ -11,11 +11,14 @@
         </vstack>
 
         <vstack style="width: 50%">
-          <typography variant="header" content="{{ time | date: 'MM/dd/yyyy' }} | {{ location }} " />
+          <hstack style="width: 100%"
+            ><typography variant="header" content="{{ time | date: 'MM/dd/yyyy' }} | {{ location }} " />
+            <latency-display style="width: 40%" />
+          </hstack>
 
           <hstack style="width: 100%">
             <viewer-display style="height: 200px; width: 33%" />
-            <latency-display style="height: 200px; width: 33%" />
+            <steering-angle-display style="height: 200px; width: 33%" />
```

Comment:
just leave the landing page alone on this PR, wanna focus on fixing it in another PR

Reply:
Ok, I needed to change it because of latency being at the top of the page. And removed old battery stuff. Maybe we could open that issue now and this will be the start of the changes.

Reply:
Hrmm, I would say just do the latency change/ removing old battery stuff in this one, my main issue was with moving the steering angle one to the top. Don't wanna arbitrarily move stuff, cause it'll p end up staying that way for a bit